aten/src/ATen/native/cuda/BatchLinearAlgebra.cpp

```diff
@@ -1850,13 +1850,7 @@ REGISTER_CUDA_DISPATCH(cholesky_inverse_stub, &cholesky_inverse_kernel_impl);
   For further details, please see the MAGMA documentation for magma_dgetrf_gpu.
 */
 template <typename scalar_t>
-static void apply_lu_looped_magma(const Tensor& input, const Tensor& pivots, const Tensor& infos, bool compute_pivots) {
-#if !AT_MAGMA_ENABLED()
```

Comment:
Why do you want to remove this `#if !AT_MAGMA_ENABLED()`?

Reply:
Note that the error did not make sense. Also, I treat all the cases of cuSOLVER / MAGMA being available or not in the calling function and I throw a reasonable error there.

Reply:
but shit, I see what you mean.